### PR TITLE
Fixed and Updated Links, grammatical errors

### PR DESCRIPTION
**Pull request by Sayanta66**
- Fixed the broken links
- Updated a deprecated link
- Fixed the grannatical errors

### CODE REVIEW

File: source/integrations/cloud-bot-accounts.rst [hunk 1]
```diff
@@ -153,7 +153,7 @@ If you find yourself unable to edit posts as a bot, check the following:
 1. Instead of using a slash command to respond directly, use an an API call for the initial interaction with a user to enable message edits.
 2. If your system is using `advanced permissions <https://docs.mattermost.com/cloud/cloud-user-management/advanced-permissions.html>`_, then post edits could be disabled for users.
 
-If neither of the above help resolve your concern, you also have the option to choose what role the bot account has. If the **System Admin** role is chosen, then they can update any posts in the system, along with other System Admin permissions. Note that giving the **System Admin** role to a bot account enables them with other System Admin privileges so this should be done with care.
+If neither of the above help resolves your concern, you also have the option to choose what role the bot account has. If the **System Admin** role is chosen, they can update any posts in the system and other System Admin permissions. Note that giving the **System Admin** role to a bot account enables them with other System Admin privileges, so this should be done with care.
```
Comment (justinegeffen) [inline source/integrations/cloud-bot-accounts.rst:156]:
```suggestion
You also have the option to choose what role the bot account has. If the **System Admin** role is chosen, they can update any posts in the system and other System Admin permissions. Note that giving the **System Admin** role to a bot account enables them with other System Admin privileges, so this should be done with care.
```

File: source/integrations/cloud-bot-accounts.rst [hunk 2]
```diff
@@ -125,12 +125,12 @@ Do bot access tokens expire?
 
 No, but you can automate your integration to cycle its token `through the REST API <https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D~1tokens%2Fpost>`_.
 
-For more information about access tokens, see `the personal access tokens documentation <https://docs.mattermost.com/cloud/cloud-integrations/cloud-personal-access-tokens.html>`_.
+For more information about access tokens, see `the personal access tokens documentation <https://docs.mattermost.com/developer/personal-access-tokens.html>`_.
```
Comment (justinegeffen) [inline source/integrations/cloud-bot-accounts.rst:128]:
```suggestion
For more information about access tokens, see `the personal access tokens documentation <https://developers.mattermost.com/integrate/admin-guide/admin-personal-access-token/>`_.
```

File: source/integrations/cloud-bot-accounts.rst [hunk 3]
```diff
@@ -2,7 +2,7 @@
 Bot Accounts
 =============
 
-Use bot accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the `personal access tokens feature <https://docs.mattermost.com/cloud/cloud-integrations/cloud-personal-access-tokens.html>`_.
+Use bot accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the `personal access tokens feature <https://docs.mattermost.com/developer/personal-access-tokens.html>`_.
```
Comment (justinegeffen) [inline source/integrations/cloud-bot-accounts.rst:5]:
```suggestion
Use bot accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the `personal access tokens feature <https://developers.mattermost.com/integrate/admin-guide/admin-personal-access-token/>`_.
```

**Review by justinegeffen — COMMENTED** [review]
Thank you!

I've updated some of the links as the integrations content has been moved to another repo. Let me know if you have any questions!

File: source/integrations/cloud-bot-accounts.rst (hunk 1 above)
Comment (Sayanta66) [reply source/integrations/cloud-bot-accounts.rst:156]: Yeah this is more apt. Less writing with more impact.

File: source/integrations/cloud-bot-accounts.rst (hunk 3 above)
Comment (Sayanta66) [reply source/integrations/cloud-bot-accounts.rst:5]: This is fine

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**